Loop/Managers/LoopDataManager.swift

```diff
@@ -1826,6 +1826,8 @@ extension LoopDataManager {
                         maximumBolus: settings.maximumBolus,
                         suspendThreshold: settings.suspendThreshold,
                         insulinSensitivitySchedule: insulinSensitivitySchedule,
-                        carbRatioSchedule: carbRatioSchedule)
+                        carbRatioSchedule: carbRatioSchedule,
+                        basalRateSchedule: basalRateSchedule,
+                        insulinModel: StoredSettings.InsulinModel(insulinModelSettings))
```

Comment:
Why StoredSettings.InsulinModel over InsulinModelSettings?

Reply:
`StoredSettings.InsulinModel` is currently available in LoopKit, but `InsulinModelSettings` isn't - is there a reason to prefer `InsulinModelSettings`?

Reply:
I think we eventually want to unify these two things... it is currently `StoredSettings.InsulinModel` because it is `Codable`.

Reply:
InsulinModelSettings will get pulled down to LoopKit with the new insulin model selection UI, so I agree it makes sense to unify then.